features/unattached_commands.feature

```diff
@@ -107,6 +107,29 @@ Feature: Command behaviour when unattached
             See: https://ubuntu.com/advantage
             """
 
+    @series.trusty
```

Comment:
It does feel a bit heavy to test this particular subcommand in both attached an unattached_commands as it costs us cycles to spin up a new container just to test this shallow non-destructive call .

Why don't we either drop this duplicated set of  tests unattached_commands or find a way to fold the `ua version` When/Then clauses into one of the other non-destructive tests like `ua refresh` on an  unattached system.